src/main/java/ch/wisv/areafiftylan/security/WebConfig.java

```diff
@@ -0,0 +1,22 @@
+package ch.wisv.areafiftylan.security;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+import javax.annotation.Nonnull;
+
+@Configuration
+public class WebConfig {
```

Comment:
What doet deze class anders dan de vorige? Toen het in `SecurityConfiguration.java`

Reply:
De `@Bean` zorgde voor een circular dependency, omdat de `SecurityConfigution` deze zelf nodig hebt door veranderingen in z'n superclass.